src/MOCK_DB_DATA/library-items/global-footer/_global-footer.data.ts

```diff
@@ -0,0 +1,9 @@
+import { LibraryItem } from 'src/app/views/shared/models/library-item.model';
+
+import { GlobalFooterPlain } from './global-footer-plain';
```

Comment:
No need to call it plain... (we need to discuss how to systemize naming of first item... in any group)

Reply:
Also renamed `-red-arch` to `-arch`